custom_components/tahoma/cover_devices/awning.py

```diff
@@ -54,8 +54,16 @@ async def async_set_cover_position(self, **kwargs):
 
     async def async_open_cover(self, **_):
         """Open the cover."""
-        await self.executor.async_execute_command(OverkizCommand.DEPLOY)
+        if self.device.device_url.startswith("rts"):
+            # Set the execution duration to 0 seconds
+            await self.executor.async_execute_command(OverkizCommand.DEPLOY, 0)
```

Comment:
Can we set this for io covers as well? Otherwise, could we create a new function `async_execute_cover_command`, that will handle this?

Reply:
It's only for rts

Reply:
This issue does not exist for io.

Reply:
But I mean, can we always pass this parameter for cover?

Reply:
I don't think so as there is no parameters for io covers. Somfy returns an error in such case no?

Reply:
Just test it, we can add this parameter to io cover too, there is no error returned.

Reply:
Great! Let's merge and test it, and than we can move it to core if no one complains!